vello_shaders/shader/flatten.wgsl

```diff
@@ -848,7 +850,7 @@ fn main(
                     let offset_tangent = offset * normalize(tangent);
                     let n = offset_tangent.yx * vec2f(-1., 1.);
                     draw_cap(path_ix, (style_flags & STYLE_FLAGS_START_CAP_MASK) >> 2u,
-                             pts.p0, pts.p0 - n, pts.p0 + n, -offset_tangent, transform);
+                        pts.p0, pts.p0 - n, pts.p0 + n, -offset_tangent, transform);
```

Comment:
This wasn't intentional - `wgsl-analyzer` has a formatter, and I use format on save. I have since disabled that formatter for wgsl files, but wasn't aware that this had been changed. I'll restore it